### PR TITLE
Switched to downloading wordpress using wp core download

### PR DESCRIPTION
**Pull request by koyan**
instead of using the registry

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -93,6 +92,7 @@
 		"reset:themes" : "rm -rf public/wp-content/themes",
 		"reset:plugins" : "rm -rf public/wp-content/plugins",
 
+		"download:wordpress": "wp core download --version=4.9.4 --path=public",
 		"copy:wordpress": "rsync -ar vendor/greenpeace/planet4-wordpress-upstream/* public",
```
Comment (kirdia) [inline composer.json]: I guess we can remove copy:wordpress completely and change docker-site-install to use download:wordpress also.

File: composer.json [hunk 2]
```diff
@@ -93,6 +92,7 @@
 		"reset:themes" : "rm -rf public/wp-content/themes",
 		"reset:plugins" : "rm -rf public/wp-content/plugins",
 
+		"download:wordpress": "wp core download --version=4.9.4 --path=public",
```
Comment (kirdia) [inline composer.json]: I don't know if we will need the --force switch here also.
What happens when we change wordpress version and the public folder already has a previous version of wordpress?

Comment (koyan) [reply composer.json]: The previous command we run is the "remove public" . So there should never be anything there.

File: composer.json (hunk 1 above)
Comment (koyan) [reply composer.json]: Correct. Doing it now.

**Pull request merged**